x-pack/platform/plugins/shared/cases/common/types/api/case/v1.ts

```diff
@@ -218,6 +218,10 @@ export const CasePostRequestRt = rt.intersection([
        * The list of custom field values of the case.
        */
       customFields: CaseRequestCustomFieldsRt,
+      /**
+       * Indicates whether the case is generated by the AI Assistant
+       */
+      isGeneratedByAssistant: rt.boolean,
```

Comment:
It was raised that there's actually no need for this to be surfaced at the api layer. A customer will//should never set this. Can it be removed from this layer?

Reply:
Thanks for pointing that out! Added adjustment commit https://github.com/elastic/kibana/pull/227338/commits/d543233f2f45fab783bf818cf5ea8787c143e5a4